waspc/data/Generator/templates/sdk/wasp/server/email/index.ts

```diff
@@ -7,24 +8,24 @@ import { EmailSender } from "./core/types.js";
 {=# isSmtpProviderUsed =}
 const emailProvider = { 
     type: "smtp",
-    host: process.env.SMTP_HOST!,
-    port: parseInt(process.env.SMTP_PORT!, 10),
-    username: process.env.SMTP_USERNAME!,
-    password: process.env.SMTP_PASSWORD!,
+    host: env.SMTP_HOST,
+    port: env.SMTP_PORT,
```

Comment:
Why did `parseInt(process.env.SMTP_PORT!, 10)` turn into `env.SMTP_PORT`? 

Because Zod not only validates that `SMTP_PORT` exists (no need for `!` anymore, here) but also casts the string to a number.